site/src/components/SearchField/SearchField.tsx

```diff
@@ -6,19 +6,28 @@ import InputAdornment from "@mui/material/InputAdornment";
 import TextField, { type TextFieldProps } from "@mui/material/TextField";
 import Tooltip from "@mui/material/Tooltip";
 import visuallyHidden from "@mui/utils/visuallyHidden";
-import type { FC } from "react";
+import { type FC, useEffect, useRef } from "react";
 
 export type SearchFieldProps = Omit<TextFieldProps, "onChange"> & {
 	onChange: (query: string) => void;
+	autoFocused?: boolean;
```

Comment:
Suggestion: `autoFocus` (action) vs `autoFocused` (past)

Reply:
renamed to `autoFocus` 👍 thanks for suggestion!